docs/AskSUSI.md

```diff
@@ -35,6 +35,26 @@ _example:_ Please explain insomnia.
 
 - Who tweeted most about `<query-term>` ? *OR* The top tweeter about `<query-term>`
 - How many times was `<query-term>` tweeted in the last 24 hours ?
+- How many tweets did `<query-username>` post in May 2016 ?
```

Comment:
This still needs an update since `<day month year>` could be used in the common query which users can make.